lib/ActivityStreams.js

```diff
@@ -33,11 +33,11 @@ function ActivityStreams(options = {}) {
   this._setupPageMod();
   this._setupListeners();
   this._setupButton();
-  this._hideAppURLs();
-  NewTabURL.override(this.options.pageURL);
+  NewTabURL.override(`${this.options.pageURL}#/`);
```

Comment:
This might not be as synchronous as you'd expect. You might want to block on obtaining the `newtab-url-changed` observer notification.

This notification is sent from https://dxr.mozilla.org/mozilla-central/source/browser/components/newtab/aboutNewTabService.js#232